### PR TITLE
Fix typo and move commands to correct clipboards.

### PR DESCRIPTION
**Pull request by Hyperkid123**
This time!

https://issues.redhat.com/browse/RHCLOUD-13714?focusedCommentId=16093477&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16093477

### CODE REVIEW

**Review by karelhala — APPROVED** [review]

**Comment by karelhala** [comment]
Let's wait for @john-dupuy approval!

**Review by john-dupuy — APPROVED** [review]

**Pull request merged**
